CONTRIBUTING.rst

```diff
@@ -109,3 +109,15 @@ Coding style
   third-party module, please consult it on GitHub before.
 - `Please use regular expressions only as a last resort. <http://imgur.com/j3G9xyP>`_ When possible, use string manipulations,
   such as :code:`split()` and then list operations. It is more readable.
+
+
+Release process
+-------------------------------------------------------------------------------
+
+1. Pick a suitable semantic version number. We adhere to `generic rules <https://docs.npmjs.com/about-semantic-versioning#incrementing-semantic-versions-in-published-packages>`_ with an exception of our `specific deprecation policy <https://github.com/tomasbedrich/pycaching/blob/master/docs/api.rst?plain=1#L7-L11>`_.
```

Comment:
@FriedrichFroebel what are your thoughts about the deprecation policy? I am wondering if we should update it to match semver, i.e. for deprecated methods to be removed together with major version bump.

Reply:
In my personal opinion, the current deprecation policy is fine and apparently no one complained about it in the past (although deprecations and releases generally are rather sparse). I would reserve major version changes for really breaking changes like dropping old Python versions or old dependency versions, which we do even less.